examples/golang/components/sqldb-postgres-query/.gitignore

```diff
@@ -2,4 +2,4 @@
 Cargo.lock
 
 # Wash build artifacts
-build
\ No newline at end of file
+buildwit/deps/
```

Comment:
`buildwit` ?

Reply:
Lol this is what happens when you script it and the .gitignore doesn't have a newline. I'll fix